drheader/core.py

```diff
@@ -282,7 +282,7 @@ def __validate_must_contain(self, config, header, directive):
                     for contain_value in config['Must-Contain']:
                         if contain_value not in cookie:
                             self.__add_report_item(
-                                severity='high' if contain_value == 'secure' else 'medium',
+                                severity=config['Severity'] if (contain_value == 'secure' and ("Severity" in config)) else 'medium',
```

Comment:
A user-provided value for severity here will only be used if `contain_value` is `secure`. For any other values, the user-provided severity will be ignored. Change to:
```
if contain_value not in cookie:
    default_severity = 'high' if contain_value == 'secure' else 'medium'
    self.__add_report_item(
        severity=config['Severity'] if ("Severity" in config) else default_severity,
```